test/e2e/bindata/assets/02_clusterrole.yaml

```diff
@@ -16,7 +16,14 @@ rules:
     resources:
       - leases
     verbs:
-      - "*"
+      - create
+      - delete
+      - deletecollection
```

Comment:
Would you mind running ci jobs by removing this. I don't think we would need this.

Reply:
No just deletecollection. I guess create and delete are mandatory